index.html

```diff
@@ -12,38 +12,49 @@
 </head>
 
 <body>
-  <div class="hero">
-    <header class="name">Escape the House</header>
-  <section class="section">
-  <div class="tile is-ancestor">
-    <div class="tile is-4 is-vertical is-parent">
-      <div class="tile is-child box">
-        <p class="title">City:</p>
+    <div class="hero">
+      <div id="header">
+
+      <header class="name">Escape the House</header>
+      </div>
+      <div class="columns">
+  <!--First column-->
+        <div class="column">
+          <div class="wrap">
+          <p class="title1">City:</p>
         <p>
-          <form class="field has-addons" id="target">
-            <div class="control" >
+          
+        <div class="field has-addons">
+          <div class="control">
             <input class="input" type="text" placeholder="..." id="cityInput" />
           </div>
           <div class="control">
             <a class="button is-info" id="searchButton"> Search </a>
           </div>
-        </form>
+        </div>
```

Comment:
I changed this to a form element so the Enter/Return keystike would work for searching. It should remain as is in the current html
```suggestion
        <form class="field has-addons" id="target">
            <div class="control" >
            <input class="input" type="text" placeholder="..." id="cityInput" />
          </div>
          <div class="control">
            <a class="button is-info" id="searchButton"> Search </a>
          </div>
        </form>
```